test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -3291,7 +3291,7 @@ private TestCpsPackageReferenceProject CreateTestCpsPackageReferenceProject(stri
                     projectId: projectName);
         }
 
-        private CpsPackageReferenceProject CreateCpsPackageReferenceProject(string projectName, string projectFullPath, ProjectSystemCache projectSystemCache)
+        internal static CpsPackageReferenceProject CreateCpsPackageReferenceProject(string projectName, string projectFullPath, ProjectSystemCache projectSystemCache)
```

Comment:
nit: Might be easier to move these out in a utility class.

Reply:
Moved factory methods to utility class.